catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/ValidationAttributes.java

```diff
@@ -37,13 +37,13 @@
         descriptors.add(new AttributeDescriptorImpl(VALIDATION_ERRORS,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
```

Comment:
This will reduce the usefulness of these attributes to facet value searches since users would not know the full text of the validation errors and warnings without searching for valid values first.